include/range/v3/view/zip_with.hpp

```diff
@@ -410,8 +410,9 @@ namespace ranges
 
             template(typename Fun)(
                 /// \pre
-                requires zippable_with<Fun>)
-            constexpr empty_view<std::tuple<>> operator()(Fun) const noexcept
+                requires zippable_with<
+                    Fun>) constexpr empty_view<detail::decay_t<invoke_result_t<Fun &>>>
```

Comment:
Formatting here, pls. If you're running this through `clang-format` you might need something like this:

```suggestion
                requires zippable_with<Fun>) //
            constexpr empty_view<detail::decay_t<invoke_result_t<Fun &>>>
```

Reply:
I don't understand why people use clang-format, exhibit N.

Reply:
To be fair, I'm really making its job unreasonably difficult with all these macros.